sprint 10.js/tema 1.js

```diff
@@ -0,0 +1,6 @@
+const num = -12;
+
+getSquare = (num) => num ** 2;
```

Comment:
```suggestion
const getSquare = (num) => num ** 2;
```
(optional) e bine sa definim fiecare functie cu const in fata.